server/aws-lsp-codewhisperer/src/language-server/chat/agents/shared/i18n.ts

```diff
@@ -0,0 +1,3 @@
+export function i18n(text: string): string {
```

Comment:
this is just a stub because we didn't know what support flare has (or is going to have) for i18n